libraries/botbuilder-core/botbuilder/core/teams/teams_info.py

```diff
@@ -103,6 +106,78 @@ class TeamsInfo:
 
         return await TeamsInfo.get_team_members(turn_context, team_id)
 
+    @staticmethod
+    async def get_paged_team_members(
+        turn_context: TurnContext,
+        team_id: str = "",
+        continuation_token: str = None,
+        page_size: int = None,
+    ) -> List[TeamsPagedMembersResult]:
+        if not team_id:
+            team_id = TeamsInfo.get_team_id(turn_context)
+
+        if not team_id:
+            raise TypeError(
+                "TeamsInfo.get_team_members: method is only valid within the scope of MS Teams Team."
+            )
+
+        connector_client = await TeamsInfo._get_connector_client(turn_context)
+        return await TeamsInfo._get_paged_members(
+            connector_client,
+            turn_context.activity.conversation.id,
+            continuation_token,
+            page_size,
+        )
+
+    @staticmethod
+    async def get_paged_members(
+        turn_context: TurnContext, continuation_token: str = None, page_size: int = None
+    ) -> List[TeamsPagedMembersResult]:
+
+        team_id = TeamsInfo.get_team_id(turn_context)
+        if not team_id:
+            conversation_id = turn_context.activity.conversation.id
+            connector_client = await TeamsInfo._get_connector_client(turn_context)
+            return await TeamsInfo._get_paged_members(
+                connector_client, conversation_id, continuation_token, page_size
+            )
+
+        return await TeamsInfo.get_paged_team_members(turn_context, team_id, page_size)
+
+    @staticmethod
+    async def get_team_member(
+        turn_context: TurnContext, team_id: str = "", member_id: str = None
+    ) -> TeamsChannelAccount:
+        if not team_id:
+            team_id = TeamsInfo.get_team_id(turn_context)
+
+        if not team_id:
+            raise TypeError(
+                "TeamsInfo.get_team_member: method is only valid within the scope of MS Teams Team."
+            )
+
+        if not member_id:
+            raise TypeError("TeamsInfo.get_team_member: method requires a member_id")
+
+        connector_client = await TeamsInfo._get_connector_client(turn_context)
+        return await TeamsInfo._get_member(
+            connector_client, turn_context.activity.conversation.id, member_id
+        )
+
+    @staticmethod
+    async def get_member(
+        turn_context: TurnContext, member_id: str
```

Comment:
Should there be a None check on member_id here?

Reply:
I would think not since the methods this calls with member_id do check.